opentelemetry-appender-tracing/src/layer.rs

```diff
@@ -173,6 +173,8 @@ where
         log_record.set_event_name(meta.name());
         log_record.set_severity_number(severity_of_level(meta.level()));
         log_record.set_severity_text(meta.level().as_str());
+        log_record.set_timestamp(std::time::SystemTime::now());
```

Comment:
As per the [specs](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/logs/data-model.md#field-timestamp) - `This field is optional, it may be missing if the source timestamp is unknown.`